WallsAndHoles/mainwindow.h

```diff
@@ -34,6 +34,9 @@ private slots:
 
     void on_actionimport_triggered();
 
+protected:
+    MeshViewContainer *mMeshViewContainer;
```

Comment:
Why is this here? Main window should remain empty. (MeshViewContainer is added as a dock widget by Editor)